framework/db/resource_manager.py

```diff
@@ -7,6 +7,7 @@
 import logging
```

Comment:
For every file, do this:

- Keep the stdlib imports first
- Followed by the OWTF internal imports

https://www.python.org/dev/peps/pep-0008/#imports
> Imports should be grouped in the following order:

>  - standard library imports
>  - related third party imports
>  - local application/library specific imports
   You should put a blank line between each group of imports.